<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>track</class>
 <widget class="QMainWindow" name="track">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="courseBox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="teeBox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>150</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmCourse">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmTeeBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QComboBox" name="startHoleBox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>250</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StartRound">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Round</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
